lib/vsc/install/commontest.py

```diff
@@ -137,10 +137,17 @@ class CommonTest(TestCase):
     ]
 
     # Prospector commandline options (positional path is added automatically)
+    cfg_path = os.path.join(os.path.dirname(__file__), '..', '..', '..', 'etc')
```

Comment:
@hajgato use `dirname` rather than `'..'`?

Also, make sure the directory exists, and that it has the file you expect; if not, produce a clear error.

Reply:
@boegel Done!